# HG changeset patch
# User Rik <rik@octave.org>
# Date 1381441967 25200
#      Thu Oct 10 14:52:47 2013 -0700
# Node ID b92242ad316f692a640a78f4f4e6f9aa9a8d5dac
# Parent  7b305b36b87e43e96c677cb38bf289676d43eb69
parser.tst: Remove comments about bug #33304 which has been fixed.

* test/parser.tst: Remove comments about bug #33304 which has been fixed.

diff --git a/test/parser.tst b/test/parser.tst
--- a/test/parser.tst
+++ b/test/parser.tst
@@ -203,31 +203,26 @@
 %! b = {1 2 3 4 5};
 %! assert (b{(a. b1 + 1)}, 4);
 %! b = 1:5;
 %! assert (b(a. b1 + 1), 4);
 %! assert ([2 3].^2', [4; 9]);
 ## Level 12 (postfix increment and decrement)
 ## No tests possible since a++-- is not valid
 ## Level 11 (transpose and exponentiation)
-## Note: Exponentiation should be left-to-right, but Octave does right-to-left.
-##       See bug #33304.
 %!test
 %! assert (2^3**2, 64);
 %! assert ([2 3].^2.', [4;9]);
 %! assert ([2 3].'.^2, [4;9]);
 %! assert (3*4i'.', 0 - 12i);
 %! assert (3*4i.'.', 0 + 12i);
 %! assert (2^-4^3, (1/16)^3);
 %! assert (2^+4^3, 16^3);
 %! assert (2^~0^2, 4);
 
-## Note: Exponentiation should be left-to-right, but Octave does right-to-left.
-##       See bug #33304.
-
 ## Level 10 (unary plus/minus, prefix increment/decrement, not)
 %!test
 %! assert (+-+1, -1);
 %! a = -1;
 %! assert (!++a, true);
 %! assert (a, 0);
 %! assert (-~a, -1);
 %! assert (!~--a, true);
@@ -282,8 +277,9 @@
 %! assert (R(pi/2), [cos(pi/2), -sin(pi/2); sin(pi/2),cos(pi/2)]);
 
 ## Check that xyz is tagged as a variable in the parser.  Both
 ## expressions must remain on one line for this test to work as
 ## intended.
 %!test
 %! xyz(1) = 1; xyz /= 1;
 %! assert (xyz, 1);
+
